packages/backend-lib/src/journeys.ts

```diff
@@ -257,13 +264,73 @@ export async function getJourneysStats({
     journeysStats.push(stats);
 
     for (const node of definition.nodes) {
+      if(node.type === JourneyNodeType.RateLimitNode || node.type === JourneyNodeType.ExperimentSplitNode) {
```

Comment:
I haven't exactly figured out what yet, after doing some local testing, I believe there's something wrong with this logic.

I created a simple test Journey, and generated some fake users, some with the `firstName: Max` in the Max segment, and others with a different name, so that some users are on both side of the segment split, and I'm never seeing anything except a 100%/0% split.

<img width="1840" alt="Screen Shot 2024-01-25 at 1 05 58 PM" src="https://github.com/dittofeed/dittofeed/assets/1750599/1ed91cf4-6ad0-4235-b205-891a87657ad8">